regimen.yml

```diff
@@ -430,7 +432,7 @@ stages:
     script: *replay_script
     script_md5: *replay_script_md5
     parameters:
-      <<: *replay_a_parameters
+      <<: *replay_b_parameters
```

Comment:
is this intentional?

Reply:
Yeah. This fixes an old bug I happened to spot. This is a 'B' stage so it should point to the b_parameters.